reports/report-manager.js

```diff
@@ -0,0 +1,226 @@
+var reportModels = require('./report-models');
+var Report = reportModels.Report;
+var EventReport = reportModels.EventReport;
+var OtherReport = reportModels.OtherReport;
+
+/**
+ * Callback used by report CRUD methods that return only one report.
+ * @callback reportCallback
+ * @param {Error} err - An error that occurred during the operation.  Null if no
+ *     errors occurred.
+ * @param {Report} report - The report operated on.
+ */
+
+/**
+ * Creates a report with the fields in the request object, and calls a callback
+ * when the database save finishes.
+ * @param {Object} reqData - The request object representing the report to be
+ *     created.
+ * @param {Object} locals - An object containing the current request's local
+ *     variables.
+ * @param {reportCallback} createCallback - Called once the operation finishes.
```

Comment:
Sorry poor wording, I meant what will be the params of the callback when it is called. But I have since found your docs for it